etc/dependency-check-suppression.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
+<!-- For information see https://jeremylong.github.io/DependencyCheck/general/suppression.html -->
+<!-- Applies to Oracle HTTP Server -->
+<suppress>
+   <notes><![CDATA[
+   file name: helidon-microprofile-server-2.1.0.jar
+   ]]></notes>
+   <packageUrl regex="true">^pkg:maven/io\.helidon\.microprofile\.server/helidon\-microprofile\-server@.*$</packageUrl>
+   <cpe>cpe:/a:oracle:http_server</cpe>
+</suppress>
+
+<!-- Applies to Processing:Processing -->
+<suppress>
+   <notes><![CDATA[
+   file name: jsonp-jaxrs-1.1.6.jar
+   ]]></notes>
+   <packageUrl regex="true">^pkg:maven/org\.glassfish/jsonp\-jaxrs@.*$</packageUrl>
+   <cpe>cpe:/a:processing:processing</cpe>
+</suppress>
+
+<!-- We use snakeyaml for reading local configuration, so this CVE
+     does not apply to our use case. For more information:
+     https://bitbucket.org/asomov/snakeyaml/issues/377/allow-configuration-for-preventing-billion
```

Comment:
We can upgrade to 1.26 that should not be affected at all (not sure if the scan knows about this). From what I see in the commit, the default behavior is not to allow recursive keys: https://bitbucket.org/asomov/snakeyaml/commits/da11ddbd91c1f8392ea932b37fa48110fa54ed8c

Reply:
Yes. See issue https://github.com/oracle/helidon/issues/2475